src/vue/entry-client.ts

```diff
@@ -51,14 +54,30 @@ export const viteSSR: ClientHandler = async function viteSSR(
     next()
   })
 
+  function redirect({ location = '/', headers = {} }: Redirection) {
```

Comment:
I don't think there should be a default redirect target. I'd rather prefer the system to crash/misbehave if incorrectly configured rather than silently redirect to `/`. This is especially true for cases when an app runs under some URL prefix such as `/myapp`. I wouldn't expect any code to redirect outside of this prefix.